generators/generator-boschiotsuite/boschiotsuite-cloud/src/main/java/org/eclipse/vorto/codegen/bosch/BoschIoTSuiteGenerator.java

```diff
@@ -1,13 +1,12 @@
 /**
- * Copyright (c) 2018 Contributors to the Eclipse Foundation
- *
- * See the NOTICE file(s) distributed with this work for additional
- * information regarding copyright ownership.
- *
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License 2.0 which is available at
- * https://www.eclipse.org/legal/epl-2.0
- *
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
```

Comment:
Suspect this would fail the plugin when building due to paragraphs and line returns :(

Reply:
should be fixed now.